unit-tests/motion/test-motion.py

```diff
@@ -0,0 +1,58 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+import pyrealsense2 as rs
+from rspy import test
+
+################################################################################################
+test.start("software-device motion stream")
+
+sd = rs.software_device()
+sensor = sd.add_sensor("Motion")
+
+intrinsics = rs.motion_device_intrinsic()
+intrinsics.data = [[1.0] * 4] * 3
+intrinsics.noise_variances = [2, 2, 2]
+intrinsics.bias_variances = [3, 3, 3]
+
+stream = rs.motion_stream()
+stream.type = rs.stream.accel
+stream.index = 0
+stream.uid = 0
+stream.fps = 200
+stream.fmt = rs.format.motion_raw
+stream.intrinsics = intrinsics
+
+stream_profile = sensor.add_motion_stream(stream).as_motion_stream_profile()
+
+sync = rs.syncer()
+
+sensor.open(stream_profile)
+sensor.start(sync)
+
+frame = rs.software_motion_frame()
+motion_frame_data = rs.vector()
+motion_frame_data.x = 1.0
```

Comment:
Can we add some "peper" and give unique values to each field?

Reply:
something like:
motion_frame_data.x = 1.0
motion_frame_data.y = 2.0
motion_frame_data.z = 3.0
is good?

Reply:
yes , maybe even
0.111
0.222
0.333
😀

Reply:
and also timetag and other 0 values

Reply:
done